bolt-boost/Cargo.toml

```diff
@@ -15,9 +18,7 @@ thiserror = "1.0.63"
 reqwest = "0.12.7"
 
 # crypto
-ssz_rs = { git = "https://github.com/ralexstokes/ssz-rs", rev = "ec3073e", features = [
-    "sha2-asm",
-] }
+ssz_rs = { git = "https://github.com/ralexstokes/ssz-rs", rev = "ec3073e" }
```

Comment:
I had to remove the ssz-rs `sha2-asm` feature as it was breaking for cross-compiled `aarch64`.
There might be a way to do it but for now I had to disable it